sockeye/lexical_constraints.py

```diff
@@ -441,12 +542,20 @@ def main():
 
         { "text": "Dies ist ein Test .", "constraints": ["This is", "test"] }
 
+    If you pass `--avoid` to the script, the constraints will be generated as negative constraints, instead:
+
+        echo -e "Dies ist ein Test .\tThis is\ttest" | python3 -m sockeye.lexical_constraints --avoid
```

Comment:
Ask the german guys, but I think "Das ist ein Test" sounds better.

Reply:
Not German but agree.